appserver/payara-appserver-modules/microprofile/metrics/src/main/java/fish/payara/microprofile/metrics/admin/MetricsServiceConfiguration.java

```diff
@@ -0,0 +1,74 @@
+/*
```

Comment:
Can you add an empty <metrics-service-configuration/> tag to the domain.xml for each config?